### PR TITLE
Well Swap

### PR DESCRIPTION
**Pull request by 0xalecks**
Adds support to do a well swap via normal sdk swap functionality

### CODE REVIEW

File: projects/examples/src/wells/liquidity.ts [hunk 1]
```diff
@@ -38,12 +37,12 @@ async function main() {
   await tx.wait();
 
   // Get LP Balance
-  const lpbal = await (await well.getLPToken()).getBalance(account);
-  // const lpbal = LPTOKEN.amount(100)
-  console.log("\nLP Balance: ", lpbal.toHuman());
+  // const lpbal = await (await well.getLPToken()).getBalance(account);
+  // // const lpbal = LPTOKEN.amount(100)
+  // console.log("\nLP Balance: ", lpbal.toHuman());
```
Comment (tansanDOTeth) [inline projects/examples/src/wells/liquidity.ts:42]: Is there a reason to preserve these comments? It's always going to be accessible in the git history if you delete it.

Comment (0xalecks) [reply projects/examples/src/wells/liquidity.ts:42]: yeah those are left commented on purpose, so you can easily do different things on your local chain by just commenting/uncommenting different parts of the files. These are just examples/local scripts used to set up tests or try things out

File: projects/examples/src/wells/swapFrom.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { WellsSDK } from "@beanstalk/wells";
+import { BeanstalkSDK, TestUtils } from "@beanstalk/sdk";
+import { signer, account, sdk as bsdk } from "../setup";
+import { TokenValue } from "@beanstalk/sdk-core";
+
+const WELL_ADDRESS = process.env.WELL_ADDRESS!;
+
+main().catch((e) => {
```
Comment (marshall-bean) [inline projects/examples/src/wells/swapFrom.ts:8]: What's the difference between `swapFrom` and `swap` example scripts?

File: projects/sdk/src/lib/swap/Swap.test.ts [hunk 3]
```diff
@@ -39,15 +39,16 @@ describe("Swap", function () {
   describe.each([
     // ETH => x
     [sdk.tokens.ETH, sdk.tokens.WETH],
-    [sdk.tokens.ETH, sdk.tokens.USDT],
-    [sdk.tokens.ETH, sdk.tokens.USDC],
-    [sdk.tokens.ETH, sdk.tokens.DAI],
-    [sdk.tokens.ETH, sdk.tokens.BEAN],
-    [sdk.tokens.ETH, sdk.tokens.CRV3],
+    // TODO: Uncomment once wells exist in fork
```
Comment (marshall-bean) [inline projects/sdk/src/lib/swap/Swap.test.ts:42]: Wdym "wells exist in fork"?

File: projects/sdk/src/constants/addresses.ts [hunk 4]
```diff
@@ -51,6 +51,11 @@ export const addresses = {
     //    - 3pool Contract            = 0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7
     Address.make("0xc9C32cd16Bf7eFB85Ff14e0c8603cc90F6F2eE49"),
 
+  // ----------------------------------------
+  // Wells Contracts
+  // ----------------------------------------
+  BEANWETH_WELL: Address.make("0xb468647b04bf657c9ee2de65252037d781eabafd"),
```
Comment (marshall-bean) [inline projects/sdk/src/constants/addresses.ts:57]: This eventually would be the production mainnet bean/weth well, right?

File: projects/sdk/src/lib/farm/LibraryPresets.ts [hunk 5]
```diff
@@ -139,5 +140,62 @@ export class LibraryPresets {
       this.bean2usdt(fromMode, FarmToMode.INTERNAL) as StepGenerator,
       this.usdt2weth(FarmFromMode.INTERNAL, toMode) as StepGenerator
     ];
+
+    this.wellWethBean = (fromToken: ERC20Token, toToken: ERC20Token, account: string, from?: FarmFromMode, to?: FarmToMode) => {
```
Comment (marshall-bean) [inline projects/sdk/src/lib/farm/LibraryPresets.ts:144]: A lot of this is hard to follow, I think we could maybe talk through it in voice for me to get a better understanding.

**Review by marshall-bean — APPROVED** [review]
